index.src.html

```diff
@@ -73,23 +76,18 @@
 <dfn>control messages</dfn>.
 
 <dfn lt="Enqueues a control message|Queue a control message">Queuing a control
-    message</dfn>
-    means adding the message to the end of a codec’s <a>control
+    message</dfn> means enqueing the message to the codec’s <a>control
```

Comment:
I'm ok with changing the prose here, but then you need to explain what enqueueing and dequeuing mean, or (and I think it would be better) to refer to https://infra.spec.whatwg.org/#queues.

My original prose in the Web Audio API text was written in a way that this was unnecessary, and predates this `infra` text.

Reply:
Done.